client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx

```diff
@@ -28,7 +30,8 @@ const InteractionsTab: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Element
     const { getDatesToInvestigate, loadInteractions, getCoronaTestDate, getClinicalDetailsSymptoms, handleDeleteContactEvent } =
         useInteractionsTab({
             setInteractions: setInteractions,
```

Comment:
You can remove the key duplication from the rest of the variables like you have done with `setAreThereContacts`: